integration/src/main/java/scripts/testscripts/GetStatus.java

```diff
@@ -1,26 +1,20 @@
 package scripts.testscripts;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
 
 import bio.terra.catalog.api.PublicApi;
 import bio.terra.testrunner.runner.TestScript;
 import bio.terra.testrunner.runner.config.TestUserSpecification;
 import com.google.api.client.http.HttpStatusCodes;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import scripts.client.CatalogClient;
 
 public class GetStatus extends TestScript {
-
-  private static final Logger log = LoggerFactory.getLogger(GetStatus.class);
-
   @Override
   public void userJourney(TestUserSpecification testUser) throws Exception {
-    log.info("Checking the service status endpoint.");
-    var publicApi = new PublicApi(new CatalogClient(server));
+    var client = new CatalogClient(server);
```

Comment:
Why move this to an intermediate variable, as opposed to accessing through `publicApi` as before? 

I think it could go either way, just curious about rationale.